lms/djangoapps/courseware/tabs.py

```diff
@@ -374,7 +375,13 @@ def get_course_tab_list(user, course):
 
     # Add in any dynamic tabs, i.e. those that are not persisted
     course_tab_list += _get_dynamic_tabs(course, user)
+    # Sorting here because although the CourseTabPluginManager.get_tab_types function
+    # does do sorting, we only use it for getting the dynamic tabs.
+    # We can't switch this function to just use the CourseTabPluginManager without
+    # further investigation since CourseTabList.iterate_displayable returns
+    # Static Tabs that are not returned by the CourseTabPluginManager.
     return course_tab_list
+    return sorted(course_tab_list, key=lambda tab: tab.priority or float('inf'))
```

Comment:
Thoughts on `float('inf')`?

Reply:
Eh, it's fine. Obvious what is happening here. Internet has some other options for doing this, but yours is just as good.

Reply:
Switched based on this find by MT https://stackoverflow.com/questions/18411560/python-sort-list-with-none-at-the-end